adafruit_sht31.py

```diff
@@ -131,7 +130,7 @@ def heater(self, value=False):
 
     @property
     def status(self):
-        """Read the sensor status."""
+        """Return the Sensor status."""
```

Comment:
Just `The sensor status`. From the outside status looks like a variable so it doesn't make sense to return anything.